docs/starlight-config.ts

```diff
@@ -10,13 +20,28 @@ export default {
     styleOverrides: {
       textMarkers: {
         defaultLuminance: ['15%', '85%'],
+        insBackground: 'var(--sl-color-green-low)',
+        insBorderColor: 'var(--sl-color-green)',
+        delBackground: 'var(--sl-color-red-low)',
+        delBorderColor: 'var(--sl-color-red)',
+        markBackground: 'var(--sl-color-purple-low)',
+        markBorderColor: 'var(--sl-color-purple)',
+      },
+      borderRadius: '0.25rem',
+      borderColor: 'var(--sl-color-border-secondary)',
+      frames: {
+        editorActiveTabIndicatorTopColor: 'var(--sl-color-accent)',
+        editorActiveTabBackground: 'var(--sl-color-bg-code)',
+        editorTabBarBackground: 'var(--sl-color-bg-secondary)',
+        editorBackground: 'var(--sl-color-bg-code)',
       },
     },
+    themes: [darkTheme, lightTheme],
   },
   title: 'Style Dictionary',
   description:
     'Export your Design Tokens to any platform. iOS, Android, CSS, JS, HTML, sketch files, style documentation, or anything you can think of. Forward-compatible with Design Token Community Group spec.',
-  logo: { src: './src/assets/logo.png', alt: 'Style-Dictionary logo, Pascal the chameleon.' },
+  logo: { src: './src/assets/logo.svg', alt: 'Style-Dictionary logo, Pascal the chameleon.' },
```

Comment:
Did you upload the SVG to this branch? When I run it locally it fails because it cannot find this asset, I reverted to PNG in my commit just FYI to make sure it doesn't error